requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
@cemsbr this is already on ``requirements-docs.txt``. By the way, why is ``Sphinx`` on ``requirements-dev.txt``?